### PR TITLE
CDHSH-71 Load profile seed data into the redux store

### PR DESCRIPTION
**Pull request by Liannus**
Changelog:
- Added functionality to fetch all the profile data and add it to the redux store, stored by ID.

Other Implementation Details:
- Created a new profile saga, slice, mountpoint and request slice.
- Added profilemodel to more easily normalize data.

### CODE REVIEW

File: services/frontend/src/utils/models/Profile.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        this.slackHandle = slackHandle;
+        this.rocketChatHandle = rocketChatHandle;
+        this.userId = userId;
+        this.createdAt = createdAt;
+        this.updatedAt = updatedAt;
+    }
+
+    /* Normalizes the list of Profiles that the API returns into a map of {ID -> Profile},
+     * with the skills processed to just their IDs, for appropriate use in the Redux store. */
+    static normalizeApiResultsForRedux(Profiles = []) {
```
Comment (DevinSit) [inline services/frontend/src/utils/models/Profile.js]: It'd have been nice if you had also copy-pasted over the test suite for this function.

File: services/frontend/src/store/slices/profiles.slice.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import {createRequestSlices} from "store/utils";
+import mounts from "store/mountpoints";
+
+export const profilesSlice = createSlice({
+    slice: mounts.profiles,
+    initialState: {},
+    reducers: {
+        setProfiles: (state, action) => action.payload,
+        addProfile: (state, action) => {
+            // Expects a Project object as payload
```
[the remark this answers is not in the document]
Comment (Liannus) [reply services/frontend/src/store/slices/profiles.slice.js]: Changed to reflect Profile object.

File: services/frontend/src/utils/models/Profile.js (hunk 1 above)
Comment (Liannus) [reply services/frontend/src/utils/models/Profile.js]: I planned to take a deeper look at the tests and commit them in my next pr, when I'm handling actually showing data on the page.

Comment (DevinSit) [reply services/frontend/src/utils/models/Profile.js]: Good enough for me :+1:

**Pull request merged**